content/blog/zephyrs-summer-intern-experience-with-pulumi/index.md

```diff
@@ -0,0 +1,81 @@
+  - I also learned to build the cloud infrastructure for a static website by configuring AWS S3 bucket, Route 53, CloudFront, and Certificate Manager. I also learned to configure AWS Fargate Service and create a Dockerfile.
+  - I learned more concepts including assume role, CDN(Content Delivery Network), target groups, and load balancers.
+
+- Besides cloud technology, I have to get in touch with the Continuous Delivery/ Continuous Integration concept and have practiced it with the Github to work with the team.
+  - I learned more details about git command including the difference between `git rebase` and `git merge`
+  - I learned to create Github projects, issues, and pull requests and link them together for better preference.
+  - I get to know about Github Actions and Workflow and implemented one for my internship project to increase efficiency in deploying.
+  - I learned about Github OAuth Applications and more about Github API while working on the OAuth Server for the CMS application.
+
+- I only learned to create a static website using plain HTML and CSS in the classroom. During my internship, I was able to get in touch with modern front-end technology including Netlify CMS, Hugo, Markdown, and React while building CMS and the preview of the website.
+
+## Reflection and Next Steps
+
+Although the basic infrastructure of the CMS application is working and provides a content management interface for the constantly updating webinar pages and events pages, there are more things I want to improve. During the last meeting with the marketing team, we have discussed creating [custom widget](https://www.netlifycms.org/docs/custom-widgets/#header) for shortcodes and image storage system. I have included in the issues and CMS Github project but don't have time to implement it.
+
+I realized I should have a clearer time schedule of development while writing the proposal and come up with a priority list of the features I want to develop in the beginning. During my meeting with the marketing team in the beginning we only discussed the potential improvement but fail to count the time it takes to develop each feature and the priority of them. If I could communicate about that better and wrote a better-timed plan, then I would be able to implement more new features.
+
+Also, as my mentor and supervisor points out, I had the bad habits of rushing through and things done without learning more details about the pathway that I choose in the beginning. If I would dig deeper into the problem, I believe I could organize the time better.
```

Comment:
```suggestion
Also, my mentor and supervisor taught me to dig deeper into the problem, so that I have a better understanding of the issue. It also helped me organize my time better when looking into issues.
```